cleanlab/util.py

```diff
@@ -294,19 +294,19 @@ def onehot2int(onehot_matrix):
 
 
 def estimate_pu_f1(s, prob_s_eq_1):
-    """Computes Claesen's estimate of f1 in the pulearning setting.
+    """Computes Claesen'labels estimate of f1 in the pulearning setting.
```

Comment:
labels -> s

Reply:
This looks like it was search-and-replaced by a bad refactoring tool (lots of `s` -> `labels` in comments)